File: src/src_core/fmg/core/img/MosaicSkillController.java
package fmg.core.img;

import fmg.core.types.ESkillLevel;

/**
 * MVC controller of {@link ESkillLevel} image
 *
 * @param <TImage> platform specific view/image/picture or other display context/canvas/window/panel
 * @param <TImageView> MVC view
 */
public abstract class MosaicSkillController<TImage,
                                            TImageView extends WithBurgerMenuView<TImage, MosaicSkillModel>>
              extends AnimatedImgController<TImage, TImageView, MosaicSkillModel>
{

   protected MosaicSkillController(boolean showBurgerMenu, TImageView imageView) {
      super(imageView);

      getView().getBurgerMenuModel().setShow(showBurgerMenu);

      usePolarLightFgTransforming(true);
      useRotateTransforming(true);
   }

}

File: src/src_core/fmg/core/img/MosaicSkillController.cs
using fmg.core.types;

namespace fmg.core.img {

   /// <summary> MVC controller of <see cref="ESkillLevel"/> image </summary>
   /// <typeparam name="TImage">plaform specific view/image/picture or other display context/canvas/window/panel</typeparam>
   /// <typeparam name="MosaicSkillModel">MVC view</typeparam>
   public abstract class MosaicSkillController<TImage, TImageView>
                       : AnimatedImgController<TImage, TImageView, MosaicSkillModel>
      where TImage : class
      where TImageView : WithBurgerMenuView<TImage, MosaicSkillModel>
   {

      protected MosaicSkillController(bool showBurgerMenu, TImageView imageView)
         : base(imageView)
      {
         View.BurgerMenuModel.Show = showBurgerMenu;

         UsePolarLightFgTransforming(true);
         UseRotateTransforming(true);
      }

   }

}
